modules/ossm-control-plane-deploy.adoc

```diff
@@ -2,38 +2,60 @@
 //
 // * service_mesh/service_mesh_install/installing-ossm.adoc
 
-[id="ossm-control-plane-deploy_{context}"]
+[id="ossm-control-plane-deploy-operatorhub_{context}"]
+= Deploying the control plane using OperatorHub
 
-= Deploying the control plane
+Follow this procedure to deploy the {ProductName} control plane from OperatorHub.
 
-With the introduction of {product-title} 4.1, the network capabilities of the host are now based on nftables rather than iptables. This change impacts the initialization of the {ProductName} application components. {ProductShortName} needs to know what host operating system OpenShift is running on to correctly initialize {ProductShortName} networking components.
+.Prerequisites
 
-[NOTE]
-====
-You do not need to follow this procedure if you are using {product-title} 4.1.
-====
+* The {ProductName} Operator must be installed.
+* Review the xref:../service_mesh_install/installing-ossm.adoc#ossm-cr-example_installing-ossm[{ProductName} custom resource] instructions.
 
-If the OpenShift installation is deployed on a Red Hat Enterprise Linux (RHEL) 7 host, then the custom resource must explicitly request the RHEL 7 `proxy-init` container image by including the following:
+.Procedure
 
-.Enabling the proxy-init container for RHEL 7 hosts
+. Log into the {product-title} web console.
 
-[subs=+macros]
-----
-  apiVersion: maistra.io/v1
-   kind: ServiceMeshControlPlane
-   spec:
-     istio:
-       global:
-        pass:quotes[*proxy_init:*]
-          pass:quotes[*image: proxy-init*]
-----
+. Create a new project named `istio-system`.
+
+. Navigate to *Catalogs* -> *Installed Operators*.
+
+. Click the {ProductName} Operator.
+
+. Under *Provided APIs*, the Operator enables you to create two resource types:
+** A {ProductName} control plane
```

Comment:
This should be "A `ServiceMeshControlPlane` resource"

Reply:
Fixed.